src/extension.ts

```diff
@@ -42,6 +43,11 @@ export async function activate(context: vscode.ExtensionContext) {
   // TODO: Do this using the graph once we migrate its dependencies onto the graph
   new CommandRunner(graph, thatMark, sourceMark);
 
+  // TODO: Remove this once tokenizer has access to graph
+  if (isTesting()) {
```

Comment:
This keeps backwards comparability with existing recorded tests. We could of course remove this and update existing tests if we so choose.